includes/class-sensei-usage-tracking.php

```diff
@@ -342,4 +335,74 @@ function handle_tracking_opt_in() {
 	function hide_tracking_opt_in() {
 		update_option( $this->hide_tracking_opt_in_option_name, true );
 	}
+
+	/**
+	 * Ensure that jQuery has been enqueued since the opt-in dialog JS depends
+	 * on it.
+	 *
+	 * @access private
+	 **/
+	function enqueue_script_deps() {
+		// Ensure jQuery is loaded
+		wp_enqueue_script( self::PREFIX . '_usage-tracking-notice', '',
+			array( 'jquery' ), null, true );
+	}
+
+	/**
+	 * Output the JS code to handle the opt-in dialog.
+	 *
+	 * @access private
+	 **/
+	function output_opt_in_js() {
+?>
+<script type="text/javascript">
+	(function( prefix ) {
+		jQuery( document ).ready( function() {
```

Comment:
If you do this instead you can use `$` instead of `jQuery` everywhere:
```
jQuery( document ).ready( function( $ ) {
```

Reply:
Nifty! I'll change that when I move it out into its own file.